packages/opal-server/opal_server/scopes/api.py

```diff
@@ -1,3 +1,4 @@
+import pathlib
```

Comment:
Are those changes here because this branch is on top of another yet-in-pr branch?